src/pages/About.jsx

```diff
@@ -20,7 +20,11 @@ const developers = [
   {
     name: 'Gonzalo Lara',
     img: testPic,
-    description: 'Todo: about Gonzalo Lara',
+    description: 'My passion for computers has lead me to pursue a career in Software '
+      + 'Engineering. I am currently a UWB CSS student completing my final year in the '
+      + 'Computer Science and Software Engineering program. I am 24 years old, bilingual '
+      + 'and currently a part of the DACA program.  I intend to contribute as much as '
+      + 'possible to building a better tomorrow.',
```

Comment:
This'll be tricky to edit later on. I recommend keeping this in a single line. If the linter complains we can disable it for these lines

Reply:
Surround the `developers` array with these lines:
```js
/* eslint-disable max-len */
const developers = [
  ...
];
/* eslint-enable max-len */
```

That'll allow the description strings to go beyond the line length.

Reply:
Ok I include it as a single line and im awaiting the lint test results if I do get an error due to that line Ill update that in the comment. Also change the surround of `developers` array with the lines you provided.

Reply:
Update, the lint error does not complain due to the line exceeding the line length and all three test passed.